docs/demultiplexing.md

```diff
@@ -160,6 +160,52 @@ Example configuration file for pipeline with `gather_from_node` specified before
 }
 ```
 
+## Dynamic batch handling with demultiplexing
+
+Demutliplexing feature enables handling requests with dynamic batch size. For workloads that involve frequent batch size change this is recommended way to handle such requests.
+In order to use it pipeline config has to have additional field demultiply_count. To leverage this feature requests, and responses have to have additional dimension of inputs/outputs.
+
+In the case of Resnet 50 with standard input (1,3,224,224) and N images, the inputs and outputs should have following shapes
+Input: (N,1,3,224,224)
+Output: (N,1,1001)
+
+Example configuration file for handling dynamic batch size:
```

Comment:
While we are adding an example, maybe it would be better to make it fully functional. Resnet was mentioned so it would be great to include a command to start ovms and run inference.
It should be also explained how the number of streams should be configured to take advantage of performance improvements.

Reply:
Example added as well with explanation of nireq & cpu streams parameters.